ferret/evaluators/faithfulness_measures.py

```diff
@@ -54,12 +54,6 @@ def compute_evaluation(
         remove_first_last, only_pos, removal_args, _ = parse_evaluator_args(
             evaluation_args
         )
-        if remove_first_last == True:
```

Comment:
Why are we removing this? In general, I'm starting to think this `remove_first_last` parameter logic should be improved (like, everywhere...)

Reply:
This was done to be consistent with the main branch, where remove_first_last will always be True by default. Here it gets set to False by hand (i.e. line 197 or in the here deleted code). We can further discuss how we manage it, keeping it True or False.

Reply:
So, with the current changes, we are actually reverting from setting it to False, correct?

Reply:
Exactly, in our changes remove_first_last is set to True by default like in the main branch and never set to False , which is done in the current task-API branch.